_style_guide/typography-list-ordered.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
The meaning are flipped. This is an [ordered list](https://www.markdownguide.org/basic-syntax/#ordered-lists):

1. They
2. Needs
3. Numbers

This is an [unordered list](https://www.markdownguide.org/basic-syntax/#unordered-lists):

 * No
 * Numbers
 * Here